libs/sdk-core/src/models.rs

```diff
@@ -847,6 +847,11 @@ pub struct ReceivePaymentResponse {
 pub struct SendPaymentRequest {
     /// The bolt11 invoice
     pub bolt11: String,
+    /// Trampoline payments outsource pathfinding to the LSP. Trampoline payments can improve
+    /// payment performance, but are generally more expensive in terms of fees and they
+    /// compromise on privacy. Setting this flag to `true` will ensure no trampoline payment is
```

Comment:
```suggestion
    /// compromise on privacy. Setting this flag to `false` will ensure no trampoline payment is
```

Reply:
Thanks, I removed that sentence, it's overly verbose now this is a 'positive' boolean.